webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -130,27 +118,27 @@ class SubscriptionsPage extends Component {
 
   async loadData() {
     const {
-      loadSetting,
       loadSubscriptions,
       loadTableColumns,
       loadTables,
       pollTasks,
       subscriptionTableSettings,
+      pingUpstreamSubscriptions,
     } = this.props;
 
     pollTasks();
-    loadSetting(CONTENT_DISCONNECTED);
     loadSubscriptions();
     await loadTables();
     loadTableColumns(subscriptionTableSettings);
+    pingUpstreamSubscriptions();
```

Comment:
This call was previously wrapped in an `isManifestImported` check which prevents us from needlessly calling this API. Did it need to move here from its original location?

Reply:
I'd prefer collect all necessary data at one point. I can revert it back if it's not a good idea.

Reply:
If it works that's OK, but I am curious about what happens when it's not wrapped in the `isManifestImported` check

Reply:
I did not notice any behavior change. `pingUpstreamSubscriptions` does not depend on `isManifestImported`.

Reply:
I think the idea behind checking `isManifestImported` is that we can reduce the # of API calls the page makes. If there is no manifest imported then there is no need to perform `pingUpstreamSubscriptions` or at least that was the case when the code was written. Since it's not related to your bugfix I see value in not changing that specific behavior. What do you think?

Reply:
Revert back to its place.